lib/plugins/aws/deploy/tests/uploadArtifacts.js

```diff
@@ -133,6 +136,19 @@ describe('uploadArtifacts', () => {
           .to.be.equal(awsDeploy.serverless.service.functions.second.artifact);
       });
     });
+
+    it('should log artifact size', () => {
+      sinon.stub(fs, 'statSync').returns({ size: 1024 });
+      sinon.stub(awsDeploy, 'uploadZipFile').returns(BbPromise.resolve());
+      sinon.spy(awsDeploy.serverless.cli, 'log');
+
+      return awsDeploy.uploadFunctions().then(() => {
+        expect(awsDeploy.serverless.cli.log.lastCall.args[0]).to.be.contain('(1 KB)');
```

Comment:
@nikgraf do you know if there is another way to do this than with `awsDeploy.serverless.cli.log.lastCall.args[0]`? This kind of makes it dependent on the last logging call being the size output, you had something different in mind recently.